src/pages/PullRequestPage/PullCoverage/routes/FileExplorer/FileExplorer.tsx

```diff
@@ -0,0 +1,49 @@
+import { useLocationParams } from 'services/navigation'
+import ContentsTableHeader from 'shared/ContentsTable/ContentsTableHeader'
+import DisplayTypeButton from 'shared/ContentsTable/DisplayTypeButton'
+import { usePullTreePaths } from 'shared/treePaths'
+import Breadcrumb from 'ui/Breadcrumb'
+import SearchField from 'ui/SearchField'
+
+import FileExplorerTable from './FileExplorerTable'
+
+import ComponentsSelector from '../ComponentsSelector'
+
+const defaultQueryParams = {
+  search: '',
+}
+
+function FileExplorer() {
+  const {
+    params,
+    updateParams,
+  }: {
+    params: { search?: string }
+    updateParams: (params: Record<string, string>) => void
+  } = useLocationParams(defaultQueryParams)
+  const { treePaths } = usePullTreePaths()
+
+  return (
+    <div className="mt-2 flex flex-col gap-2">
+      <ContentsTableHeader>
+        <div className="flex items-center gap-4">
+          <DisplayTypeButton />
+          <Breadcrumb paths={treePaths} />
+        </div>
+        <div className="flex gap-2">
+          <ComponentsSelector />
```

Comment:
Does not look like we are rendering the flags selector here.

We want both flags and components selector if they exist. hence why I used ToggleHeader with `noBottomBorder` in the deleted jsx file. 

We also want them to be inline next to each other 

<img width="1378" alt="Screenshot 2024-04-10 at 3 03 52 PM" src="https://github.com/codecov/gazebo/assets/148245014/c631cd69-90f9-4dbe-81d7-310c6ba19e43">